torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4342,6 +4373,21 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.fmin,
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_max_min_binary,),
+    OpInfo('fmod',
+           dtypes=all_types_and(torch.float16, torch.bool),
+           sample_inputs_func=sample_inputs_fmod_remainder,
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_jit'),
+               SkipInfo('TestCommon', 'test_variant_consistency_eager'),
```

Comment:
`test_variant_consistency_jit_fmod_cuda_float32`, `test_variant_consistency_jit_fmod_cpu_float32` tests were failing, which I'm not really sure why. I have skipped these tests for now, any help on this will be appreciation.

```
FAIL: test_variant_consistency_jit_fmod_cuda_float32 (__main__.TestCommonCUDA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_utils.py", line 1033, in wrapper
    method(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_utils.py", line 1033, in wrapper
    method(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 292, in instantiated_test
    result = test_fn(self, *args)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 266, in test_wrapper
    return test(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/test/test_ops.py", line 438, in test_variant_consistency_jit
    self.assertAutodiffNode(traced_fn.last_graph, op.assert_autodiffed, nonfusible_nodes, fusible_nodes)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_jit.py", line 275, in assertAutodiffNode
    self.assertEqual(should_autodiff_node,
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_utils.py", line 1391, in assertEqual
    super().assertTrue(x == y, msg=msg)
AssertionError: False is not true : 
Failure in testing nodes' autodifferentiation. One or more nodes were not expected to be autodiffed but were found in a DifferentiableGraph or in a FusionGroup of a DifferentiableGraph. Did you intend for these nodes to be autodiffed? If so, change this test to expect autodifferentiation. 
Specifically:
  ['aten::fmod'] were not expected to be in one of the DifferentiableGraphs but were.
```